js/help_options.js

```diff
@@ -1,84 +1,26 @@
-    },
-    {
-        "text" : "Canvas 24x7 chat",
-        "subtext" : "Live chat with Canvas Support",
-        "url": "https://cases.canvaslms.com/apex/liveagentchat"
-    },
-    {
-        "text" : "Canvas 24x7 email support",
-        "subtext" : "Email Canvas Support for a quick reply",
-        "url": "#create_ticket"
-    },
-    {
-        "text" : "Canvas 24x7 hotline",
-        "subtext" : "Call Canvas Support at 1-844-326-4466 for immediate assistance",
-    },
-    {
-        "text" : "Canvas Guides",
-        "subtext" : "Find answers to common questions",
-        "url": "https://community.canvaslms.com/community/answers/guides/"
-    },
-];
-
-/*
-    add the new items to the menu
-*/
-function addNewHelpItems(menuOptions){
-
-    $.each(menuOptions, function(idx, obj) {
-        if (obj.url) {
-            $('#help-dialog-options').append(
-                $('<li>').append(
-                    $('<a>')
-                        .attr('href', obj.url)
-                        .attr('target', '_blank')
-                        .append($('<span>').attr('class', 'text').append(obj.text))
-                        .append($('<span>').attr('class', 'subtext').append(obj.subtext))
-                )
-            );
-        } else {
-            $('#help-dialog-options').append(
-                $('<li>').append(
-                    $('<span>')
-                        .attr('class', 'info-no-url')
-                        .append($('<span>').attr('class', 'text').append(obj.text))
-                        .append($('<span>').attr('class', 'subtext').append(obj.subtext))
-                )
-            );
-        }
-    });
+function removeUnwantedHelpOptions() {
+  $('div#help_tray ul li a:contains(Ask the Community)').parent().hide();
+  $('div#help_tray ul li a:contains(Submit a Feature Idea)').parent().hide();
 }
 
 /*
     The help options do not exist on the page until the help button is clicked and the ensuing ajax call returns.
-    The observer will what the page and look for new nodes that match the help-dialog selector. When one is found
-    the method will empty the list, add the new help options, and display the menu to the user.
+    The observer will watch the page and look for new nodes that match the help_tray selector. When one is found
+    the method will remove the unwanted list items.
 */
 function initHelpMenu(){
 
     MutationObserver = window.MutationObserver || window.WebKitMutationObserver;
     var helpMenuObserver = new MutationObserver(function (mutations) {
         mutations.forEach(function (mutation) {
             var $target = $(mutation.target);
-            if ($target.find('#help-dialog-options').length != 0) {
-                $('#help-dialog-options').empty();
-                addNewHelpItems(menuOptions);
-                $('#help-dialog-options').css('display', 'block');
+            if ($target.find('#help_tray').length != 0) {
+                removeUnwantedHelpOptions()
```

Comment:
Actually - it turns out that the contents of the help menu is reloaded each time you open it, so we do need to leave the observer connected.